src/style-spec/reference/v8.json

```diff
@@ -4392,6 +4392,27 @@
         "data-driven styling": {}
       }
     },
+    "raster-scaling": {
+      "type": "enum",
+      "doc": "The raster scaling to use for overscaling",
+      "values": {
+        "linear": {
+          "doc": "(Bi)linear filtering"
+        },
+        "nearest": {
+          "doc": "Nearest neighbor filtering"
+        }
+      },
+      "default": "linear",
+      "function": "piecewise-constant",
```

Comment:
You'll need to rebase this PR to reflect the changes in https://github.com/mapbox/mapbox-gl-js/pull/6521 (which will replace the `function` and `zoom-function` keys here) (sorry for the delay in reviewing!)

Reply:
rebased

Reply:
Ah, you'll also need to mirror that updated taxonomy in this property — this is what's causing tests to fail. Delete the `zoom-function` and `function` keys, and add `property-type` and `expression`:
``` diff
13,14c13,17
<       "function": "piecewise-constant",
<       "zoom-function": true,
---
>       "property-type": "data-constant",
>       "expression": {
>         "interpolated": false,
>         "parameters": ["zoom"]
>       },

```

Reply:
ah I see now, yeah due to changes which landed after I submitted the PR. Thanks for the pointers.